### PR TITLE
update 1829

### PR DESCRIPTION
**Pull request by volker18xxdev**
100% Ownership, upgrade towns to cities, insert draft-round without Bids

### CODE REVIEW

**Comment by volker18xxdev** [comment]
this offense:
Offenses:

queue.rb:27:18: C: [Correctable] Performance/RegexpMatch: Use match? instead of =~ when MatchData is not used.
    next true if user.email =~ /@(msn hotmail outlook live passport)/
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
doesn't come not from my files.

**Comment by Zwergenpunk** [comment]
The problem with the rubocop should be fixed. Just rebase and push again :)

**Comment by volker18xxdev** [comment]
done

File: lib/engine/game/g_1829/Step/buy_sell_par_shares.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  module Game
+    module G1829
+      module Step
+        class BuySellParShares < Engine::Step::BuySellParShares
+          def actions(entity)
+            return [] unless entity == current_entity
+
+            actions = super
+
+            actions << 'pass' unless actions.include?('pass') || actions.empty?
```
Comment (tobymao) [inline lib/engine/game/g_1829/Step/buy_sell_par_shares.rb:15]: don't use unless with multiple conditions

File: lib/engine/game/g_1829/Step/close_company.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G1829
+      module Step
+        class CloseCompanyVoluntary < Engine::Step::BuyTrain
+          def process_close_company(_action)
+            @log << "#{company.name} is closed."
```
Comment (tobymao) [inline lib/engine/game/g_1829/Step/close_company.rb:9]: why do you need 2 logs? just have one

**Comment by tobymao** [comment]
the Step/draft should not be capitalized

**Comment by volker18xxdev** [comment]
please give a hint, which file? (i'll copied the draft file from 18CZ, so i don't know, where is the mistake.)

**Pull request merged**